creme/compat/sklearn.py

```diff
@@ -382,7 +382,7 @@ def predict(self, X):
         """
 
         # Check the fit method has been called
-        utils.validation.check_is_fitted(self)
+        utils.validation.check_is_fitted(self, attributes="instance_")
```

Comment:
Can you use single quotes? Apart from that the PR is good :)

Reply:
Done !